docs/src/tutorial/example-data.rst

```diff
@@ -0,0 +1,89 @@
+Run the workflow using example data
+===================================
+
+In this tutorial, you will run the workflow using example reference data.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`setup`. These instructions will install all of the software you need to complete this tutorial and others.
+
+Setup
+-----
+
+1. Activate the ``nextstrain`` conda environment:
+
+   .. code:: text
+
+      conda activate nextstrain
+
+2. Change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+3. Download the example tutorial repository into a new directory ``ncov-tutorial/``:
+
+   .. code:: text
+
+      $ git clone https://github.com/nextstrain/ncov-tutorial
+      Cloning into 'ncov-tutorial'...
+      [...more output...]
+
+Run the workflow
+----------------
+
+From within the ``ncov/`` directory, run the ``ncov`` workflow using a configuration file provided in the tutorial directory:
+
+.. code:: text
+
+   nextstrain build . --cores 4 --configfile ncov-tutorial/example-data.yaml
+
+Break down the command
+~~~~~~~~~~~~~~~~~~~~~~
+
+The workflow can take several minutes to run. While it is running, you can learn about the parts of this command:
```

Comment:
Just for our own reference, the example build ran in 7 minutes with 3 CPUs. The refine step ran for 4 min 40 seconds of that total time.